test/utils/bytecode.hpp

```diff
@@ -297,6 +297,17 @@ inline bytecode calldataload(bytecode index)
     return index + OP_CALLDATALOAD;
 }
 
+inline bytecode calldatasize()
+{
+    return OP_CALLDATASIZE;
+}
+
+inline bytecode calldatacopy(
+    bytecode src = push(0), bytecode dst = push(0), bytecode size = calldatasize())
```

Comment:
I wouldn't use defaults, seems useful to see explicitly what goes into the code (on the other hand calls and creates already have implicit defaults)

Reply:
Done.